docs/storage/prometheus.md

```diff
@@ -14,79 +14,79 @@ To monitor cAdvisor with Prometheus, simply configure one or more jobs in Promet
-`container_cpu_schedstat_runqueue_seconds_total` | Counter | Time duration processes of the container have been waiting on a runqueue | seconds
-`container_cpu_system_seconds_total` | Counter | Cumulative system cpu time consumed | seconds
-`container_cpu_usage_seconds_total` | Counter | Cumulative cpu time consumed | seconds
-`container_cpu_user_seconds_total` | Counter | Cumulative user cpu time consumed | seconds
-`container_file_descriptors` | Gauge | Number of open file descriptors for the container |
-`container_fs_inodes_free` | Gauge | Number of available Inodes |
-`container_fs_inodes_total` | Gauge | Total number of Inodes |
-`container_fs_io_current` | Gauge | Number of I/Os currently in progress |
-`container_fs_io_time_seconds_total` | Counter | Cumulative count of seconds spent doing I/Os | seconds
-`container_fs_io_time_weighted_seconds_total` | Counter | Cumulative weighted I/O time | seconds
-`container_fs_limit_bytes` | Gauge | Number of bytes that can be consumed by the container on this filesystem | bytes
-`container_fs_reads_bytes_total` | Counter | Cumulative count of bytes read | bytes
-`container_fs_reads_total` | Counter | Cumulative count of reads completed |
-`container_fs_read_seconds_total` | Counter | Cumulative count of seconds spent reading |
-`container_fs_reads_merged_total` | Counter | Cumulative count of reads merged
-`container_fs_sector_reads_total` | Counter | Cumulative count of sector reads completed
-`container_fs_sector_writes_total` | Counter | Cumulative count of sector writes completed
-`container_fs_usage_bytes` | Gauge | Number of bytes that are consumed by the container on this filesystem | bytes
-`container_fs_write_seconds_total` | Counter | Cumulative count of seconds spent writing | seconds
-`container_fs_writes_bytes_total` | Counter | Cumulative count of bytes written | bytes
-`container_fs_writes_merged_total` | Counter | Cumulative count of writes merged |
-`container_fs_writes_total` | Counter | Cumulative count of writes completed |
-`container_hugetlb_failcnt` | Counter | Number of hugepage usage hits limits |
-`container_hugetlb_max_usage_bytes` | Gauge | Maximum hugepage usages recorded | bytes
-`container_hugetlb_usage_bytes` | Gauge | Current hugepage usage | bytes
+Metric name | Type | Description | Unit (where applicable) | -disable_metrics parameter
+:-----------|:-----|:------------|:------------------------|:-------
+`container_accelerator_duty_cycle` | Gauge | Percent of time over the past sample period during which the accelerator was actively processing | percentage | accelerator 
+`container_accelerator_memory_total_bytes` | Gauge | Total accelerator memory | bytes | accelerator
+`container_accelerator_memory_used_bytes` | Gauge | Total accelerator memory allocated | bytes | accelerator
+`container_cpu_cfs_periods_total` | Counter | Number of elapsed enforcement period intervals | | cpu
```

Comment:
cpu option is not visible in cAdvisor help,
see:
```
  -disable_metrics metrics
        comma-separated list of metrics to be disabled. Options are 'accelerator', 'cpu_topology','disk', 'diskIO', 'network', 'tcp', 'udp', 'percpu', 'sched', 'process', 'hugetlb', 'referenced_memory'. (default sched,process,hugetlb,referenced_memory,cpu_topology,tcp,udp,advtcp)
```

Reply:
These metrics cannot be disabled. I removed these entries.